compiler/passes/insertLineNumbers.cpp

```diff
@@ -287,9 +303,8 @@ LineAndFile InsertLineNumbers::getLineAndFileForFn(FnSymbol *fn) {
     DefExpr* bundleArg = toDefExpr(fn->formals.tail);
     AggregateType* bundleType = toAggregateType(bundleArg->sym->typeInfo());
 
-    VarSymbol* lineField = newTemp("_ln", dtInt[INT_SIZE_DEFAULT]);
-    bundleType->fields.insertAtTail(new DefExpr(lineField));
-
+    VarSymbol* lineField = getOrCreateField(bundleType, "_ln",
```

Comment:
May be worth adding a comment here indicating how this field could have already been created.

Seeing this I was thinking oh maybe we'll end up redundantly creating `lineLocal` (and the assignment) as well but I guess the thing is (as it was before) we ended up mutating the type twice since it was shared across specializations but the other code isn't.

Reply:
> he type twice since it was shared across specializations but the other code isn't.

Yeah. There's one arg bundle that's used for two `coforall_fn`s. `lineLocal` is needed per `coforall_fn`, `lineField` is needed per arg bundle.